### ISSUE TITLE
Marks PHP7 return types as a syntax error

### ISSUE THREAD

**Issue by woozy** [AtomLinter/linter-php#235]
I'm using phpmd 2.6.

Is this supposed to happen? 

![image](https://cloud.githubusercontent.com/assets/356706/22802185/c8af0576-ef07-11e6-9de1-7180904b5f3b.png)

**Comment by Arcanemagus** [comment]
`phpmd` has nothing to do with this, that's a separate package. Can you verify that the `php` in your `$PATH` (or the path explicitly specified in your settings) is actually v7 and not an older version?

**Comment by woozy** [comment]
@Arcanemagus, yes I have php v7.1. That's Why I thought this could be related with the plugin:

`sergio@deathstar:~$ php -v
PHP 7.1.0 (cli) (built: Dec  2 2016 09:19:56) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.1.0-dev, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.1.0, Copyright (c) 1999-2016, by Zend Technologies`

**Comment by Arcanemagus** [comment]
Can you run through these steps:
* Open developer tools
* Find the code for this package
* Place a breakpoint [here](https://github.com/AtomLinter/linter-php/blob/v1.3.1/lib/main.js#L35)
* Reload Atom (View -> Developer -> Reload Window)

It should pause at that breakpoint, and the `output` variable should contain the version string of whatever `php` Atom is using.

**Comment by michaeldzjap** [comment]
I ran into the same issue. I'm on macOS Sierra. Running `php -v` from the command line I get PHP 7.1.0. However, when I go to the steps suggested above I get `output: "PHP 5.6.28 ...` in the Atom console. I tried adding  the lines 

```
"linter-php":
  executablePath: "/usr/local/bin/php"
```

to my config.cson file (result from executing `which php` from the OS terminal), but that didn't seem to make a difference. Then I tried just setting the executable path via Settings/Packages/linter-php/Settings and that did the trick for me. Hope that helps.
